substrate/frame/revive/src/exec.rs

```diff
@@ -547,6 +549,8 @@ struct Frame<T: Config> {
 	read_only: bool,
 	/// The caller of the currently executing frame which was spawned by `delegate_call`.
 	delegate_caller: Option<Origin<T>>,
+	/// The output of the last executed call frame.
+	last_frame_output: ExecReturnValue,
```

Comment:
Shouldn't it be an `Option` so we are forced to explicitly handle the cases when there was no last call? How do the opcodes behave when there was no last sub call?

Reply:
I think an option only complicates things? The "return data" is a bytes array and if there is no last call yet, there is nothing special about it, it just has a length of 0. So if this was an option we would just do a pointless `unwrap_or_default()` everywhere we access it?

Reply:
Yes I thought maybe we have to trap in case you call those opcodes but apparently copying zero bytes is what they do. So this is fine.

Reply:
Yeah thats what I found too 👍